### PR TITLE
fix man-page to mention changed default DPI setting for PNGs export

### PR DESCRIPTION
**Pull request by Jojo-Schmitz**
follow up for 2d83656c (so since MuseScore 3.4), which changed that from 300 to 72.0 * 5

### CODE REVIEW

File: mscore/musescore.cpp [hunk 1]
```diff
@@ -7473,21 +7473,21 @@ MuseScoreApplication::CommandLineParseResult MuseScoreApplication::parseCommandL
...
-      parser.addOption(QCommandLineOption("score-transpose", "Transposes the given score and exports the data to a single JSON file, prints it to std out", "options"));
-      parser.addOption(QCommandLineOption("raw-diff", "Print a raw diff for the given scores"));
-      parser.addOption(QCommandLineOption("diff", "Print a diff for the given scores"));
+      parser.addOption(QCommandLineOption(      "score-media", "Export all media (excepting mp3) for a given score in a single JSON file and print it to stdout"));
+      parser.addOption(QCommandLineOption(      "score-meta", "Export score metadata to JSON document and print it to stdout"));
+      parser.addOption(QCommandLineOption(      "score-mp3", "Generates mp3 for the given score and export the data to a single JSON file, print it to stdout"));
+      parser.addOption(QCommandLineOption(      "score-parts-pdf", "Generates parts data for the given score and export the data to a single JSON file, print it to stdout"));
+      parser.addOption(QCommandLineOption(      "score-transpose", "Transposes the given score and exports the data to a single JSON file, print it to stdout", "options"));
```
Comment (mirabilos) [inline mscore/musescore.cpp:7488]: `prints` because all other verbs are also in this form

Comment (Jojo-Schmitz) [reply mscore/musescore.cpp:7488]: I see "Print...", "Use...", "Start...", "Dump...", "Export...", "Set...", "Specify", "Load...", "Execute...", "Save...", "Revert...", "Process...", "Enable...", "Override...", "Run...", "Don't use...", "Install...".
So the majority is not, but "Used with...", "Generates...", "Transposes", so needs fixing.

**Comment by mirabilos** [comment]
>So the majority is not, but "Used with...", "Generates...", "Transposes", so needs fixing.

Ah, I looked in the context, which has Generates, Transposes, etc.

OK fix it the other way round then ☻

**Comment by Jojo-Schmitz** [comment]
still needed for master too

**Pull request merged**